src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -3034,6 +3044,16 @@ operand immI_0()
 %}
 
 // Constant for increment
+operand immI1()
```

Comment:
This operand should not be re-introduced.

Reply:
Done.